deployment-examples/aws/serverless/extension/README.md

```diff
@@ -1,5 +1,14 @@
 ## Deploy as AWS Lambda
 
-1. For deployment to AWS Lambda, zip the extensions folder and specify `src/lambda.handler` as the entry point for the AWS Lambda function.
-2. When the extension module is run as AWS Lambda **it will NOT create** the required resources like custom types or commercetools API extension for you. 
+1. Run 'npm run zip-lambda-function'. It helps to execute following steps
+   - Rename index.lambda.js to index.js
+   - For deployment to AWS Lambda, bundle the following files and folders into a .zip file and upload it in AWS console.
+      - src
+      - node_modules
+      - resources
+      - index.js
+      - package.json
+2. Specify `index.handler` as the entry point for the AWS Lambda function.
```

Comment:
Has it been tested in AWS for this naming already?
For the GCP case, it is specified as 'extensionTrigger' and it doesn't require to be named as 'index.extensionTrigger'.

Reply:
It's different than GCP:
<img width="261" alt="Screenshot 2021-02-15 at 19 55 17" src="https://user-images.githubusercontent.com/3469524/107984229-c76e4600-6fc7-11eb-9a04-ffa6544a5254.png">